src/emit.ts

```diff
@@ -30,7 +30,7 @@ export function emitted<T = unknown>(
 export const attachEmitListener = () => {
   events = {}
   // use devtools to capture this "emit"
-  setDevtoolsHook(createDevTools(events))
+  setDevtoolsHook(createDevTools(events), this)
```

Comment:
I _think_ `this` is the proper value to pass here

Reply:
Ho right, sorry I missed that. Opened a PR to fix it https://github.com/vuejs/vue-test-utils-next/pull/1011